pkg/apis/kubeone/validation/validation_test.go

```diff
@@ -738,7 +738,7 @@ func TestValidateCloudProviderSpec(t *testing.T) {
 	for _, tc := range tests {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			errs := ValidateCloudProviderSpec(tc.providerConfig, nil)
+			errs := ValidateCloudProviderSpec(tc.providerConfig, kubeoneapi.ClusterNetworkConfig{}, nil)
```

Comment:
Please add some tests for the case you added.

Reply:
done